package.json

```diff
@@ -7,15 +7,16 @@
     "doc": "docs"
   },
   "scripts": {
-    "speccy": "speccy serve api/swagger.json",
-    "prod:server": "NODE_ENV=production ts-node --files --transpile-only ./server/index.ts",
     "dev:server": "ts-node --files  --transpile-only --project ./tsconfig.server.json ./server/index.ts",
-    "dev": "./node_modules/nodemon/bin/nodemon.js --watch 'server/**/*.ts' --exec 'npm run dev:server'",
-    "start": "npm run prod:server",
-    "nodemon": "./node_modules/nodemon/bin/nodemon.js",
+    "dev": "npx nodemon --watch 'server/**/*.ts' --exec 'npm run dev:server'",
```

Comment:
any particular reason to npx nodemon instead of using the one installed as a dep?

Reply:
In this case npx gets the nodemon binary from the node_module folder (installed deps) hence I find using npx cleaner than calling the binary manually.

Reply:
Oh neat I didn't know it worked like that - thanks! Otherwise lgtm, will merge when at computer

Reply:
Ah can you resolve the conflicts? Probably just new packages

Reply:
Resolved them